tests/__snapshots__/test_options.ambr

```diff
@@ -31,7 +31,7 @@
     line-height: normal;
     margin-left: 10px;
     margin-right: 10px;
-    color: lightred;
+    color: rebeccapurple;
```

Comment:
I don't have enough context in the PR description to understand what this means or why it is important

Reply:
The `"lightred"` color is not a valid CSS color name (and not a web safe color). So using it results in an error. I instead opted for `"rebeccapurple"` since it's a valid CSS3 color name (and not a valid X11 color name).